app/Console/Commands/SetupCommand.php

```diff
@@ -0,0 +1,101 @@
+{
+    use ConfiguresApplication;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'chompy:setup {--reset}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Configure your application.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->createEnvironmentFile($this->option('reset'));
+
+        $this->section('Set Northstar environment variables', function () {
+            $environments = [
+                'http://northstar.dev' => 'http://aurora.dev',
+                'https://identity-dev.dosomething.org' => 'https://aurora-qa.dosomething.org',
+                'https://identity-qa.dosomething.org' => 'https://aurora-thor.dosomething.org',
+            ];
+
+            $this->chooseEnvironmentVariable('NORTHSTAR_URL', 'Choose a Northstar environment', array_keys($environments));
+
+            $this->instruction('You can get these environment variables from Aurora\'s "Clients" page:');
+            $this->instruction($environments[env('NORTHSTAR_URL')] . '/clients');
+
+            $this->setEnvironmentVariable('NORTHSTAR_AUTHORIZATION_ID', 'Enter the OAuth Client ID');
+            $this->setEnvironmentVariable('NORTHSTAR_AUTHORIZATION_SECRET', 'Enter the OAuth Client Secret');
+            $this->setEnvironmentVariable('NORTHSTAR_CLIENT_ID', 'Enter the Northstar Client ID');
+            $this->setEnvironmentVariable('NORTHSTAR_CLIENT_SECRET', 'Enter the Northstar  Client Secret');
+        });
+
+
+        $this->section('Set Rogue environment variables', function () {
+            $environments = [
+                'https://rogue-qa.dosomething.org' => 'https://aurora-qa.dosomething.org',
+                'https://rogue-thor.dosomething.org' => 'https://aurora-thor.dosomething.org',
+            ];
+
+            $this->chooseEnvironmentVariable('Rogue_URL', 'Choose a Rogue environment', array_keys($environments));
+
+        });
+
+        $this->section('Set Driver environment variables', function () {
+            $this->setEnvironmentVariable('BROADCAST_DRIVER', 'Enter the broadcast driver');
+            $this->setEnvironmentVariable('CACHE_DRIVER', 'Enter the cache driver');
+            $this->setEnvironmentVariable('SESSION_DRIVER', 'Enter the session driver');
+            $this->setEnvironmentVariable('QUEUE_DRIVER', 'Enter the queue driver');
+
+        });
+
+        $this->section('Set Pusher environment variables', function () {
+            $this->setEnvironmentVariable('PUSHER_APP_ID', 'Enter the Pusher App ID');
+            $this->setEnvironmentVariable('PUSHER_APP_KEY', 'Enter the Pusher App Key');
+            $this->setEnvironmentVariable('PUSHER_APP_SECRET', 'Enter the Pusher App Secret');
+
+        });
+
+        $this->section('Set S3 environment variables', function () {
```

Comment:
I think we can remove this, and maybe add a comment that the `FILESYSTEM_DRIVER` will determine if S3 is used to store files or if the local storage driver would be used.